src/Octoshift/Services/ReclaimService.cs

```diff
@@ -173,7 +173,8 @@ public virtual async Task ReclaimMannequins(string[] lines, string githubTargetO
                 continue;
             }
 
-            var result = await _githubApi.ReclaimMannequin(githubOrgId, userid, claimantId);
+            var result = skipInvitation ? await _githubApi.ReclaimMannequinsSkipInvitation(githubOrgId, userid, claimantId) : await _githubApi.ReclaimMannequin(githubOrgId, userid, claimantId);
```

Comment:
I would refactor/rename `ReclaimMannequin` to what it is actually doing `CreateAttributionInvitation` so it is clear what this is actually calling. The original name is misleading.